build/timestamp.gni

```diff
@@ -28,4 +28,7 @@ if (is_official_build) {
 # changes to keep this up to date.  (Bots run gn on each build, and for devs
 # the timestamp being 100% accurate doesn't matter.)
 # See compute_build_timestamp.py for tradeoffs for picking the timestamp.
-build_timestamp = "123"
+build_timestamp = exec_script(compute_build_timestamp,
```

Comment:
In order for reproducible builds to work, we don't want to put a timestamp in our build—if the test is solely checking this build_timestamp value, I think we should disable the test

Reply:
That is what the test is trying to test.

> we don't want to put a timestamp in our build

So is this a permanently or temporarily decision? If we just don't want to put a timestamp in our build, I agree we can just disable the test.

Reply:
It's a permanent decision, at least for now. Let's just disable the test